src/main/mw.c

```diff
@@ -236,14 +237,14 @@ void annexCode(void)
     }
 
     if (feature(FEATURE_VBAT)) {
-        if ((int32_t)(currentTime - vbatLastServiced) >= VBATINTERVAL) {
+        if (cmp32(currentTime, vbatLastServiced) >= VBATINTERVAL) {
```

Comment:
Yeah that's better I think. It's a really nice way of handling subtraction of uints producing a negative result, but just hard to come up with a good name for it. :stuck_out_tongue: